### ISSUE TITLE
onClick and onChange doesn't fire on radio buttons on iOS

### ISSUE THREAD

**Issue by badtant** [facebook/react#9956]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
onClick and onChange doesn't fire on radio buttons on iOS
TypeError: undefined is not an object (evaluating 'descriptor.get.call')

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via https://jsfiddle.net or similar (template: https://jsfiddle.net/84v837e9/).**
```
<input type="radio" name="vote" value="0" onChange={this._sendVote} />
<input type="radio" name="vote" value="1" onChange={this._sendVote} />
```
                                    
**What is the expected behavior?**
onClick and onChange should fire

**Which versions of React, and which browser / OS are affected by this issue? Did this work in previous versions of React?**
React 15.6.0 on iOS. Was working on 15.5.4

**Comment by gaearon** [comment]
cc @aweary @nhunzaker

**Comment by gaearon** [comment]
also cc @jquense, this code was added in https://github.com/facebook/react/pull/8575.

**Comment by jquense** [comment]
o my there should have been a fix in with this...perhaps it only got into master? let me find it

**Comment by jquense** [comment]
arg, the problem with so much time between these, I forgot that this was also added to master: https://github.com/facebook/react/pull/6648/files

**Comment by gaearon** [comment]
Thanks! Do you remember if there was anything else, or we can cut 15.6.1 with this?

**Comment by jquense** [comment]
Should be it, at least in terms of changes i've made. I didn't hear about any other issues so I don't think there was anything else added...

**Comment by flarnie** [comment]
I'll cherry-pick it and cut 15.6.1 now. Thanks @jquense and @gaearon ~

**Comment by flarnie** [comment]
Just an update - we're waiting to cut 15.6.1 until a fix is in for https://github.com/facebook/react/issues/9963. Will release it either way within a day.

**Comment by gaearon** [comment]
Should be fixed in 15.6.1. Please verify.

**Comment by bbnnt** [comment]
not sure if the issue is related, but having this error on safari desktop (version 9.1.1); both with react 15.6.0 and 15.6.1

**Comment by gaearon** [comment]
Please provide a minimal example (e.g. on jsfiddle) that consistently reproduces the issue with 15.6.1 and Safari 9.1.1.

**Comment by badtant** [comment]
@gaearon my reported bug is confirmed fixed

**Comment by JulianSuringa** [comment]
I  experience this error I replicate in safari 9.1 react version 15.6.0 and 15.6.1

**Comment by toshisugano** [comment]
I also experience this in Safari 9.1.2

**Comment by gaearon** [comment]
It doesn’t help to comment on closed issues. In most cases people don’t look at them because the old problem was solved. When you start experiencing an issue similar to an old issue, please **file a new one** so that the maintainers can keep track of it, and provide all the necessary details.

It appears that someone did: https://github.com/facebook/react/issues/10755. If you could share more information about the versions you use, and the reproducing cases there, that would be splendid!

**Comment by JulianSuringa** [comment]
Confirm fix safari 9.1.3 for react version 5.6.1 and react-dom 15.6.1.
